cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less

```diff
@@ -262,22 +261,34 @@
         border-top: 1px solid @gray-10;
         border-bottom: 1px solid @gray-10;
         margin: unit( 20px / @base-font-size-px, em )
-            unit( 20px / @base-font-size-px, em )
-            unit( 20px / @base-font-size-px, em )
-            0;
+                unit( 20px / @base-font-size-px, em )
+                unit( 20px / @base-font-size-px, em )
+                0;
 
         &.geolocating {
-            background: transparent 
-                        url( '@{image_path}icon_spinner.gif' ) 
-                        scroll 
-                        no-repeat 
-                        center 
+            background: transparent
+                        url( '@{image_path}icon_spinner.gif' )
+                        scroll
+                        no-repeat
+                        center
                         center;
             #chart {
                 opacity: 0;
             }
         }
 
+        .highcharts-color-0 {
+            fill: #ADDC91;
```

Comment:
TRIVIAL: at some point we'll probably want this and the fill below to be lowercase as that's the setting we have in CF stylelint, which may come over here at some point https://github.com/cfpb/capital-framework/blob/canary/.stylelintrc#L3